installer/PowerToysSetup/Resources.wxs

```diff
@@ -213,6 +213,15 @@
         </RegistryKey>
         <File Id="GcodePreviewHandler_$(var.IdSafeLanguage)_File" Source="$(var.BinDir)\$(var.Language)\PowerToys.GcodePreviewHandler.resources.dll" />
       </Component>
+      <Component
+          Id="QoiPreviewHandler_$(var.IdSafeLanguage)_Component"
+          Directory="Resource$(var.IdSafeLanguage)INSTALLFOLDER"
+          Guid="$(var.CompGUIDPrefix)08">
```

Comment:
Oops, this was my bad, missed that you put this in the middle of the components list, i.e. the next Component *Launcher_Calculator_$(var.IdSafeLanguage)_Component* has (var.CompGUIDPrefix)08 guid. I'd suggest moving this to the end of the components list and Guid should be Guid="$(var.CompGUIDPrefix)1E" if I'm not wrong

Reply:
Done!